lib/flame/controller.rb

```diff
@@ -177,11 +177,27 @@ def default_path
 			## @example Inherit controller with parent actions by method
 			##   class MyController < BaseController.with_actions
 			##   end
-			def with_actions
+			## @example Include actions from module in controller
+			##   class MyController < BaseController
+			##     include with_actions Module1
+			##     include with_actions Module2
+			##     ....
+			##   end
+			def with_actions(mod = nil)
+				return mod.extend(IncludingActions) if mod
 				@with_actions ||= Class.new(self) { extend ParentActions }
 			end
 		end
 
+		## Extension to include modules for re-defining methods controller methods
+		module IncludingActions
+			def included(ctrl)
+				public_instance_methods.each do |meth|
+					ctrl.send :define_method, meth, public_instance_method(meth)
+				end
+			end
+		end
```

Comment:
`.public_instance_method(public_method)` is repeated. Just `superclass` or `self`, I guess.

`define_method public_method, um` is also repeated. Just `self` or `ctrl`, I guess.